src/extractors/custom/otrs.com/index.js

```diff
@@ -0,0 +1,43 @@
+export const OtrsComExtractor = {
+  domain: 'otrs.com',
+
+  title: {
+    selectors: [['meta[name="og:title"]', 'value']],
+  },
+
+  author: {
+    selectors: ['div.dateplusauthor a'],
+  },
+
+  date_published: {
+    selectors: [['meta[name="article:published_time"]', 'value']],
+  },
+
+  dek: {
+    selectors: [['meta[name="og:description"]', 'value']],
+  },
+
+  lead_image_url: {
+    selectors: [['meta[name="og:image"]', 'value']],
+  },
+
+  content: {
+    selectors: ['#main article'],
+
+    defaultCleaner: false,
+
+    transforms: {
+      'div.blockquote-inner': $node => {
+        const text = $node.text();
+        $node.html(`<strong>"${text}"</strog>`);
```

Comment:
There's a type here, should be `</strong>`. (Just saw this by accident)

Reply:
Thank you!